apps/voting/app/src/components/VoteText.js

```diff
@@ -16,25 +16,34 @@ const VoteText = React.memo(
 
     return (
       <TextComponent>
-        {text.split('\n').map((line, i) => (
-          <React.Fragment key={i}>
-            {transformAddresses(line, (part, isAddress, index) =>
-              isAddress ? (
-                <span title={part} key={index}>
-                  {' '}
-                  <LocalIdentityBadge
-                    badgeOnly={disabled}
-                    compact
-                    entity={part}
-                  />{' '}
-                </span>
-              ) : (
-                <span key={index}>{part}</span>
-              )
-            )}
-            <br />
-          </React.Fragment>
-        ))}
+        <span
+          css={`
+            a {
```

Comment:
Which component is this selecting? If it's the badge, we should be able to style it directly?